tests/snapshots/semgrep-core/84cced48c6d1/stdxxx

```diff
@@ -0,0 +1,120 @@
+--- begin input files ---
```

Comment:
I think Martin added the option to specify where the snapshot is located, should we start using that already so it's easy to map tests with their snapshots?

Reply:
In the end I'm doing some Logs.app calls to add the origin of the test in the snapshot, but yes
maybe explicit path to the snapshot would be nice, but then we need to find a way to not
repeat ourselves ... the nice thing with pytest is that it derives the location automatically from the 
test name. It can be annoying some times but in practice it's pretty nice.

Reply:
I think the explicit use of snapshot.assert_match(result, "results.txt")
is also pretty nice; Right now our test just say ~checked_output:Testo.stdxxx
but without any explicit reference to a snapshot file, so you don't really understand the test, that
the actual assert is outside and its content defined in a snapshot buried under tests/snapshots/semgrep-core/ab244556/

Reply:
But for now I'm fine with storing the test origin in the snapshot. Better than nothing.